src/files.test.ts

```diff
@@ -43,8 +53,20 @@ describe("testing files functions", () => {
 
     if (files1.value) {
       expect(files1.value).toHaveLength(2);
-      expect(files1.value[0]?.name).toBe("file1.txt");
-      expect(files1.value[1]?.name).toBe("file2.txt");
+      expect(files1.value[0]).toBeDefined();
+      if (files1.value[0]) {
```

Comment:
Why bother with this `if(...` ? The prior `expect` will have already crapped out if this isn't true, unless there's some weird supported semantic where `files1.value[0]` is defined but is falsey

Reply:
Oh that sucks. OK leave as-is for now. It's not really any worse than @ts-ignore ;-)